@here/harp-examples/src/datasource_object-picking.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import { CopyrightElementHandler, MapView, PickResult } from "@here/harp-mapview";
+import { APIFormat, OmvDataSource } from "@here/harp-omv-datasource";
+import { accessToken } from "../config";
+
+/**
+ * This example showcases how picking works.
+ *
+ * For enable line picking set `enableRoadPicking: true` in [[MapViewOptions]] and set
+ * `createTileInfo: true` in [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ * For enable polygon picking set `gatherFeatureIds: true` in
+ * [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ * For enable text element picking set `gatherFeatureIds: true` in
+ * [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ *
+ * Now, let's write an event that fires when the user clicks the map canvas:
+ * ```typescript
+ * [[include:datasource_geojson_picking1.ts]]
```

Comment:
the name of the file is different, I would align it.

Reply:
done